app/src/pages/inside/settingsPage/settingsTabs/notificationsTab/notificationsTab.jsx

```diff
@@ -72,21 +84,28 @@ export class NotificationsTab extends Component {
 
     return { ...config, emailCases };
   }
+  isOnMobile = () => window.matchMedia(SCREEN_XS_MAX).matches;
+  toggleReadOnlyMode = () => {
+    debounce(() => {
+      const isMobile = this.isOnMobile();
+      const isAbleToEditForm = this.isAbleToEditForm();
+      this.setState({ editable: isAbleToEditForm && !isMobile });
```

Comment:
I think no make sense to create variables which uses just one time.
`this.setState({ editable: this.isAbleToEditForm() && !this.isOnMobile() });` - still pretty readable.

Reply:
Fixed